isp/api_channels.go

```diff
@@ -1026,7 +1026,7 @@ type ApiPatchChannelRequest struct {
 	ifNoneMatch *[]string
 	ifModifiedSince *time.Time
 	ifUnmodifiedSince *time.Time
-	patchChannelRequest2Inner *[]PatchChannelRequest2Inner
+	patchChannelRequestInner *[]PatchChannelRequestInner
```

Comment:
There are a number of small breaking changes like this one in this PR. Seems that maybe the order of something was maybe scrambled? I see a lot of files being renamed from 1 to 2 or 2 to 1. It may be a case of needing to re-generate into the right order if it's random, otherwise digging in to figure out what is causing these renames.